sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/FeatureFlagConfigurationSetting.java

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.models;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * {@link FeatureFlagConfigurationSetting} model.
```

Comment:
The link is pointing to this class itself. Add more details to this JavaDoc. This is a slightly advanced feature and should be well documented on how and why this model should be used.